pg_search/tests/pg_regress/expected/layer_size_config.out

```diff
@@ -0,0 +1,42 @@
+\i common/common_setup.sql
+CREATE EXTENSION IF NOT EXISTS pg_search;
+-- Disable parallel workers to avoid differences in plans
+SET max_parallel_workers_per_gather = 0;
+SET enable_indexscan to OFF;
+SET paradedb.enable_mixed_fast_field_exec = true;
+DROP TABLE IF EXISTS layer_sizes;
+CREATE TABLE layer_sizes (id serial8 not null primary key);
+-- 1 layer ✅
+CREATE INDEX idxlayer_sizes ON layer_sizes USING bm25(id) WITH (key_field='id', foreground_layer_sizes = '1kb');
+DROP INDEX idxlayer_sizes;
+CREATE INDEX idxlayer_sizes ON layer_sizes USING bm25(id) WITH (key_field='id', background_layer_sizes = '1kb');
+DROP INDEX idxlayer_sizes;
+-- negative layer ❌
+CREATE INDEX idxlayer_sizes ON layer_sizes USING bm25(id) WITH (key_field='id', foreground_layer_sizes = '-1kb');
+ERROR:  a single layer size must be non-negative
+DROP INDEX idxlayer_sizes;
+ERROR:  index "idxlayer_sizes" does not exist
```

Comment:
I know this is the output file, but there's no need for this test to try and drop an index it knows shouldn't exist.  It's a little confusing as I asked myself "why"

Reply:
Looks like this test follows this pattern a number of times...